### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by Berkay-akbas**
## Changes
- This branch was created because I was getting linters setup error in github actions even though locally I had no problem.
- For more detailed commit history you can check this [pr](https://github.com/Berkay-akbas/crypto-metrics/pull/4)
- When the page loads, the web app shows the list of categories listed by coins market cap.
- Each item has the current price next to it.
- You can filter the coins by searching using the search bar or text-to-voice.
- On the details page, data is retrieved from API and showed further coin statistics.
- Tests are implemented for each component and actions.
- Project deployed on the live server using netlify.

for the video link describing the project [click here](https://www.loom.com/share/67fd97accbfd46dc8c155957558720fe).

### CODE REVIEW

File: src/components/Textvoice.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        onTouchEnd={(e) => mouseUp(e)}
+        onMouseUp={(e) => mouseUp(e)}
+        type="button"
+        icon={faMicrophone}
+        className="fontawesome"
+      />
+    </div>
+  );
+};
+export default Dictaphone;
```
Comment (rotimiazeez) [inline src/components/Textvoice.js:61]: - Great job you did here 👏🏽. It was nice implementing the `textvoice` in your project, keep up the good work 😉.

File: src/tests/Components.test.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    .create(
+      <Provider store={store}>
+        <Router>
+          <Navbar />
+        </Router>
+      </Provider>,
+    )
+    .toJSON();
+  expect(NavbarTest).toMatchSnapshot();
+});
```
Comment (rotimiazeez) [inline src/tests/Components.test.js:45]: - Please note that some of your tests are failing, kindly fix them 👍🏽
Screenshot from 2022-09-01 04-16-39

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @Berkay-akbas  👏🏽, 
## Good job so far! 👍🏽 
### Summary ♨️ 
- No Linter error ✔️ 
- Good and descriptive PR 💯 
- Good README 📖
- Good video presentation( Kindly pay more attention you look a bit distracted) 👍🏽
- Great UI 🖼️
You still need to work on some issues to go to the next project, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/tests/Components.test.js (hunk 2 above)
Comment (rloterh) [reply src/tests/Components.test.js:45]: This is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Berkay,  

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Highlights 👍
- API fetch properly implemented
- Navigation working as intended
- State management with Redux properly implemented
- Good unit test coverage


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You can also consider:**
- I think you should change the header title _Author_ to _Authors_, as this project is worked on by two contributors. 
- MIT license link is broken, kindly endeavor to fix it
Implementing the above changes will make your ReadMe appear more professional.
<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**